README.md

```diff
@@ -54,31 +43,30 @@ cd /home/rstudio/sc-rna-seq-snap/analyses/<module_of_interest>
 <img width="650" alt="how-to-fork-repo-3" src="https://github.com/user-attachments/assets/073abb78-3993-4527-a574-859fd3046d39">
 
 
-4. Replace the `project_parameters.Config.yaml` with your file paths and parameters.
+## To run the code in this repository:
 
-5. How to run a module analysis
+1. Replace the `project_parameters.Config.yaml` with your file paths and parameters.
 
-Before running a module, you will do `sync fork` of your project repo at GitHub, if your branch is behind the main branch of the `stjude-dnb-binfcore/sc-rna-seq-snap:main`. This will update the main branch of your project repo with the new code and modules (if any). This will add code and not break any analyses already run in your project repo. 
+2. Navigate to an analysis module and run the shell script of interest:
+```
+cd ./sc-rna-seq-snap/analyses/<module_of_interest>
+```
 
-Then navigate to your `./sc-rna-seq-snap` project repo and ensure you are at the `main` branch. If not, you will need to `git checkout` to the main branch.
+**If you have forked the repo**, you will need to do the following steps before running the script of interest. **If you have cloned the repo, you can skip this.**
+
+You need to do `sync fork` of your project repo at GitHub before running a module, if your branch is behind the main branch of the `stjude-dnb-binfcore/sc-rna-seq-snap:main`. This will update the main branch of your project repo with the new code and modules (if any). This will add code and not break any analyses already run in your project repo. 
 
+Then navigate to your `./sc-rna-seq-snap` project repo and ensure you are at the `main` branch. If not, you will need to `git checkout` to the main branch.
 ```
 git branch
 git checkout main
 ```
 
 Finally, `git pull` to get the most updated changes and code in your project repo. 
-
 ```
 git pull
```

Comment:
If I sync and do a git pull for the forked repo after updating the yaml file to my paths, I get an error saying I either need to commit or stash (or restore) the changes to my yaml before completing the pull. So maybe either the info for syncing and pulling the forked repo should be moved back up above the section to run the code in the repo. Or else maybe add instructions for how to handle the yaml file.

Reply:
This has been addressed here 314de0925e8d728e2e313a1c80ee90118bd519bf.